### PR TITLE
Show role selector for education users

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Show role selector for education users

If the user has not yet selected a role
Show the role selector after one day

Refactor unit tests

[stage-2]

## Motivation and Context
Collect company role for Education customers. Part of UX Improvements epic.

## How Has This Been Tested?
Tested role collection and update scenarios in the local environment. Validated with unit tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

File: web/scripts/common-header/controllers/ctr-company-role-modal.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      $scope.user = user;
+      $scope.COMPANY_ROLE_FIELDS = EDUCATION_COMPANY_ROLE_FIELDS;
+
+      $scope.save = function () {
+        $modalInstance.close({
+          user: user
+        });
+      };
+
+      $scope.selectRole = function (roleValue) {
```
Comment (ezequielc) [inline web/scripts/common-header/controllers/ctr-company-role-modal.js]: @alex-deaconu I couldn't find where `$scope.selectRole()` is being used. Is it dead code? Thanks!

Comment (alex-deaconu) [reply web/scripts/common-header/controllers/ctr-company-role-modal.js]: @ezequielc good question. I just copied from https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/common-header/controllers/ctr-company-icp-modal.js#L21-L27. Looks like that isn't being used either.

Will remove both.

Thanks.

Comment (ezequielc) [reply web/scripts/common-header/controllers/ctr-company-role-modal.js]: @alex-deaconu Yeah, I think we should remove. Thanks!

Comment (alex-deaconu) [reply web/scripts/common-header/controllers/ctr-company-role-modal.js]: @ezequielc updated, thanks!

Comment (ezequielc) [reply web/scripts/common-header/controllers/ctr-company-role-modal.js]: @alex-deaconu Awesome. Thanks!

**Pull request merged**